src/Post/RegisteredTypesScope.php

```diff
@@ -13,9 +13,9 @@
 
 use Illuminate\Database\Eloquent\Builder;
 use Illuminate\Database\Eloquent\Model;
-use Illuminate\Database\Eloquent\ScopeInterface;
+use Illuminate\Database\Eloquent\Scope;
 
-class RegisteredTypesScope implements ScopeInterface
+class RegisteredTypesScope implements Scope
```

Comment:
Scopes have changed a bit in 5.2. See https://laravel.com/docs/5.3/upgrade#upgrade-5.2.0 > Eloquent > Global Scopes.

I think we can get rid of the `remove` method, and all of the weird whereIndex/bindingIndex/bindingCount stuff that makes it possible.

Please test post type functionality after doing this to make sure it works.

Reply:
Done. Tested.